dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/Param.tsx

```diff
@@ -11,6 +11,7 @@ interface IParamProps {
   appValues: string;
   param: IBasicFormParam;
   id: string;
+  otherParams: IBasicFormParam[];
```

Comment:
nitpick: The name `otherParams` implies that the property excludes the current param but it's not excluding it. Probably `allParams` is a better name.

Reply:
That's true, totally agree.